flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/config/ConsumerConfigConstants.java

```diff
@@ -134,6 +134,10 @@ public SentinelSequenceNumber toSentinelSequenceNumber() {
 	/** The interval between each attempt to discover new shards. */
 	public static final String SHARD_DISCOVERY_INTERVAL_MILLIS = "flink.shard.discovery.intervalmillis";
 
+	/** The config to turn on adaptive reads from a shard. */
+	public static final String SHARD_USE_ADAPTIVE_READS = "flink.shard.use.adaptive.reads";
```

Comment:
[most Flink's feature flags](https://ci.apache.org/projects/flink/flink-docs-master/ops/config.html) are named `xx.enabled`,  I'd suggest rename it to something like `flink.shard.adaptive.read.records.enabled`

Reply:
Changed.